cpp/src/io/comp/io_uncomp.h

```diff
@@ -42,7 +42,7 @@ enum {
 
 std::vector<char> io_uncompress_single_h2d(const void* src, size_t src_size, int stream_type);
 
-std::vector<char> get_uncompressed_data(host_span<char const> const& data,
+std::vector<char> get_uncompressed_data(host_span<char const> const data,
```

Comment:
Not that is really matters, but we don't mark parameters passed by value as const (and Jake's suggestion aligns with this).
```suggestion
std::vector<char> get_uncompressed_data(host_span<char const> data,
```

Reply:
I left the `const` intentionally so it's known those variables never change. One less thing to think about.